apps/frontend/src/pages/Profile.tsx

```diff
@@ -0,0 +1,255 @@
+import {
+  Avatar,
+  Box,
+  Button,
+  TextField,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+import AccountSidebar from "../components/AccountSidebar";
+import { useStyles } from "../styles/pageStyles";
+import { ChangeEvent, useEffect, useState } from "react";
+import _axios_instance from "../_axios_instance";
+import { useParams } from "react-router-dom";
+
+interface ImageURLObject {
+  url: string;
+}
+
+const Profile = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.down("lg"));
+  const { id } = useParams();
+
+  const [, setImage] = useState<File | null>(null);
```

Comment:
did this run?
```suggestion
  const [image, setImage] = useState<File | null>(null);
```

Reply:
My bad, I actually don't need that variable